client/fresheyes-bot/src/util.ts

```diff
@@ -65,6 +65,19 @@ export function getIssueBody<T extends Record<string, any>>(arg: T) {
   return { body };
 }
 
+export function generateIssueBody<T extends Array<Record<string, any>>>(arg: T) {
+  const authors = Array.from(new Set(arg.map((item) => item?.user?.login))).length;
```

Comment:
We can simplify this logic by calling the `.size` property on the Set. This eliminates the need to convert the set back to an array just to get the length.

Reply:
Alright will make necessary corrections for this